### PR TITLE
Add support for enabling repo metadata GPG signing

### PR DESCRIPTION
**Pull request by PaulSD**
See https://github.com/theforeman/puppet-certs/pull/188

### CODE REVIEW

File: templates/yum_importer.json.erb [hunk 1]
```diff
@@ -19,9 +19,5 @@
     <% else %>
     "proxy_password": null,
     <% end %>
-    <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::real_yum_max_speed']) -%>
-    "max_speed": <%= scope['pulp::real_yum_max_speed'] %>"
-    <% else %>
-    "max_speed": null
-    <% end %>
```
Comment (stbenjam) [inline templates/yum_importer.json.erb]: Did you intend to remove this?

Comment (PaulSD) [reply templates/yum_importer.json.erb]: Oops.  Good catch.

File: manifests/init.pp [hunk 2]
```diff
@@ -188,6 +188,8 @@
 #
 # $yum_max_speed::              The maximum download speed for a Pulp task, such as a sync. (e.g. "4 Kb" or 1Gb")
 #
+# $gpg_sign_repo_metadata::     Whether repo metadata GPG signing will be enabled
```
Comment (ekohl) [inline manifests/init.pp]: Slight nit: should this perhaps be ```yum_gpg_sign_repo_metadata``` or at least mention it in the documentation?

Comment (PaulSD) [reply manifests/init.pp]: Ok, I don't have a strong opinion, so I'll change it.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**